exasol/exaslpm/model/package_file_config.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+from pydantic import BaseModel, FilePath
+
+class PhaseLevels(Enum):
+    Phase1 = 'Phase 1'
+    Phase2 = 'Phase 2'
+
+class Installer(Enum):
+    Pip = 'pip'
+    Apt = 'apt'
+    Conda = 'conda'
+
+class DependencyLevels(Enum):
+    UdfClient = 'udfclient_deps'
+    Language = 'language_dpes'
+    Build = 'build_deps'
+    BuildTest = 'build_test_deps'
+    Flavor = 'flavor_base_deps'
+
+class Packages(BaseModel):
+    name: str
+    version: str
+    bin_path: FilePath
```

Comment:
Currently, we inject the binary path from the outside. Furthermore, we wouldn't define it for each single package but for a whole phase.